ui/src/components/facets/HistogramFacet.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import VegaLite from "react-vega-lite";
+import { Handler } from "vega-tooltip";
+
+const styles = {};
+
+const baseSpec = {
+  $schema: "https://vega.github.io/schema/vega-lite/v3.json",
+  mark: "bar",
+  width: 400,
+  height: 300,
+  encoding: {
+    color: {
+      field: "dimmed",
+      type: "nominal",
+      scale: {
+        range: ["#4c78a8", "#aaafb7"]
+      },
+      legend: null,
+    },
+    tooltip: [
```

Comment:
Replace 21-32:
```
    tooltip: [
      {
        title: "facet_value",
        field: "count",
        type: "quantitative",
      },
    ],
```
I want tooltip to be "female: 1760"

Reply:
Done.

I was only able to do this by adding a 'text' field to the data with this format. Unfortunately the [tooltip spec](https://vega.github.io/vega-lite/docs/encoding.html#text) doesn't allow you use two fields in the title like that.